docs/content/staking/custody-providers.mdx

```diff
@@ -0,0 +1,78 @@
+on Flow.
+
+## Blocto
+
+For details about how to use your Blocto account to stake and earn rewards you'll
+find everything you need to know in Blocto's [staking guide](https://guide.blocto.app/article/stake-flow-tokens-step-by-step-with-blocto).
+
+#### Withdrawing Rewards
+
+Users using the Blocto wallet can also withdraw their rewards using the Flow Port!
+
+---
+
+## Ledger
+
+If you're storing your locked FLOW tokens using a Ledger device, you can use the Flow Port to **stake** or **delegate** them to earn rewards.
+For more details you can read [our guides](https://docs.onflow.org/support/flow-port/) to familiarize yourself with the Flow Port app,
+and how to use it as a staking portal.
+
+#### Withdrawing Rewards
+
+Use [Flow Port](https://port.onflow.org) to withdraw any rewards you earned by staking or delegating locked FLOW form your Ledger device.
+
+---
+
+## Coinlist
+
+Coinlist is operating FLOW nodes and will automatically delegate your locked FLOW to them as stake.
+If you would like to withdraw your rewards, you'll need to contact Coinlist's support team.
+
+#### Withdrawing Rewards
+
+If you would like to withdraw your rewards, you'll need to contact Coinlist's support team.
+
+---
+
+## Kraken
+
+Kraken is operating FLOW nodes and will automatically delegate your locked FLOW to them as stake.
+For details about how to delegate, or withdraw your rewards, you'll need to contact Kraken's support team by email.
+
+#### Withdrawing Rewards
+
+If you would like to withdraw your rewards, you'll need to contact Kraken's support team.
+
+---
+
+## Finoa (Coming Soon!)
+
+Instructions for user's holding their locked tokens with Finoa will be provided soon. If you have questions please contact Finoa's support team directly.
+
+## Summary
+
+| Provider | Auto-restaking | Custom Staking/Delegation | Stake Owned Nodes | Stake Custom Nodes | Flow Port Integration |
+| -------- | -------------- | ------------------------- | ----------------- | ------------------ | --------------------- |
+| Bocto    | 🚫             | ✅                        | ✅                | 🚫                 | ✅                    |
+| Ledger   | 🚫             | ✅                        | ✅ \*             | 🚫                 | ✅                    |
+| Coinlist | ✅             | 🚫                        | ✅                | 🚫                 | 🚫                    |
+| Kraken   | ✅             | 🚫                        | 🚫                | ✅                 | 🚫                    |
+| Finoa    |                |                           |                   |                    |                       |
+
+### Legend
+
+- **Auto-restaking**: The porvider will automatically re-stake any rewards you earn!
+- **Custom delegation**: The provider allows token holders to decide where to delegate their stake
+- **Stake owned nodes**: The provider will stake nodes they are running themselves
+- **Stake custom nodes**: The provider will stake nodes they choose, but may not own
```

Comment:
The provider will stake nodes they choose, but may not own  -> The provider will delegate user tokens to nodes they choose, but may not own